scripts/create_releases_pr.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -e
+
+export DOCS_SITE_BASE_DIR="."
+export GITHUB_ORG_NAME=gruntwork-io
+
+export RELEASES_PLUGIN_BEFORE_DATE=
+export BRANCH_NAME=
+export CONFIGURE_GIT=${CONFIGURE_GIT:=1}
+
+function set_release_date() {
+  unameOut="$(uname -s)"
+
+  # Handling running this on linux or mac
+  if [ ${unameOut} = "Linux" ]; then
+    RELEASES_PLUGIN_BEFORE_DATE=$(date -d "$(date +"%Y-%m-01") - 1 day" +"%F")
+  elif [ ${unameOut} = "Darwin" ]; then
+    RELEASES_PLUGIN_BEFORE_DATE=$(date -v1d -v-1d +%Y-%m-%d)
+  else
+    echo "Unknown uname ${unameOut}, exiting"
+    exit 1
+  fi
+}
+
+function configure_git {
+  local -r git_username="$1"
+  local -r git_email="$2"
+  git config --global user.name "$git_username"
+  git config --global user.email "$git_email"
+  git config --global pull.rebase false
+}
+
+function set_branch_name() {
```

Comment:
consider whether you want to change to:
- Branch name isn't a global env var
- This function is `get_branch_name` which returns the formatted branch name
- You pass that as a param into the functions that need it.

It's not a big deal - but i found it weird that there was a global looking variable at the top that was not initialized.

Since this isn't shared with anything..nor is it any kind of reusable component, I don't think it _really_ matters..so this whole thing is just a nit.

Reply:
Updated.